test/solc/CommandLineParser.cpp

```diff
@@ -388,10 +388,10 @@ BOOST_AUTO_TEST_CASE(standard_json_mode_options)
 
 	CommandLineOptions expectedOptions;
 	expectedOptions.input.mode = InputMode::StandardJson;
-	expectedOptions.input.paths = {};
-	expectedOptions.input.standardJsonFile = "input.json";
+	expectedOptions.input.paths = {"input.json"};
 	expectedOptions.input.basePath = "/home/user/";
 	expectedOptions.input.allowedDirectories = {"/tmp", "/home", "project", "../contracts"};
+	expectedOptions.input.ignoreMissingFiles = true;
 	expectedOptions.output.dir = "/tmp/out";
```

Comment:
It was already there, but it should never be hardcoded to `/tmp`. Is this used and written to somewhere? (if not, ignore this message). :)

Reply:
`CommandLineParser` is not meant to read anything from disk so it's fine. Note that all the actual file reading and checks for path existence are in `CommandLineInterface`. This is for example the reason why both have separate conditions to check if there were any input files - even if `CommandLineParser` checks it and sees that there's at least one path, `CommandLineInterface` with `--ignore-missing` could still see zero files because they do not actually exist.

I decided to use these paths like this because dealing with absolute paths when you need to ensure they exist overcomplicates tests a lot. See for example tests like `cli_paths_to_source_unit_names_relative_base_path` from `test/solc/CommandLineInterface.cpp` in #11545. These tests would be much simpler if I could just write paths like this but there I can't.